lib/livebook/runtime/evaluator/default_formatter.ex

```diff
@@ -128,4 +128,17 @@ defmodule Livebook.Runtime.Evaluator.DefaultFormatter do
   defp error_color(string) do
     IO.ANSI.format([:red, string], true)
   end
+
+  defp error_type(error) do
+    cond do
+      system_env_error?(error) -> {:unavailable_env, error.env}
+      true -> :other
+    end
+  end
+
+  defp system_env_error?(exception) do
+    is_struct(exception, System.EnvError) and
+      Exception.message(exception) =~
+        "could not fetch environment variable"
```

Comment:
@josevalim a sanity check, should we match on the message or is the exception for missing envs by definition?